syntax = "proto3";

package fintekkers.requests.price;

import "fintekkers/models/price/price.proto";
import "fintekkers/requests/price/query_price_request.proto";

option java_multiple_files = true;

option java_outer_classname = "QueryPriceResponseProtos";

message QueryPriceResponseProto {
  string object_class = 1;
  string version = 2;

  fintekkers.requests.price.QueryPriceRequestProto query_price_request = 20;

  repeated fintekkers.models.price.PriceProto price_response = 30;
}


